jumanji/environments/routing/multi_cvrp/utils.py

```diff
@@ -0,0 +1,296 @@
+        late_coef_rand,
+        customer_demand_max,
+    )
+
+
+def get_20_customer_init_settings(
+    num_vehicles: int,
+) -> Tuple[chex.Array, chex.Array, chex.Array, chex.Array, chex.Array, chex.Array]:
+    if num_vehicles not in [2, 3]:
+        raise ValueError("num_vehicles must be 2 or 3 for num_customers=20")
+    map_max = 10
+    max_capacity = 60
+    max_start_window = 10.0
+    early_coef_rand = (0.0, 0.2)
+    late_coef_rand = (0.0, 1.0)
+    if num_vehicles == 2:
+        customer_demand_max = 10
+    else:
+        customer_demand_max = 15
+
+    return (
+        map_max,
+        max_capacity,
+        max_start_window,
+        early_coef_rand,
+        late_coef_rand,
+        customer_demand_max,
+    )
+
+
+def get_50_customer_init_settings(
+    num_vehicles: int,
+) -> Tuple[chex.Array, chex.Array, chex.Array, chex.Array, chex.Array, chex.Array]:
+    if num_vehicles not in [2, 3, 4, 5]:
+        raise ValueError("num_vehicles must be 2, 3, 4 or 5 for num_customers=50")
+    map_max = 20
+    max_capacity = 150
+    max_start_window = 20.0
+    early_coef_rand = (0.0, 0.2)
+    late_coef_rand = (0.0, 1.0)
+    if num_vehicles == 2:
+        customer_demand_max = 10
+    elif num_vehicles == 3:
+        customer_demand_max = 15
+    elif num_vehicles == 4:
+        customer_demand_max = 20
+    else:
+        customer_demand_max = 25
+    return (
+        map_max,
+        max_capacity,
+        max_start_window,
+        early_coef_rand,
+        late_coef_rand,
+        customer_demand_max,
+    )
+
+
+def get_100_customer_init_settings(
+    num_vehicles: int,
+) -> Tuple[chex.Array, chex.Array, chex.Array, chex.Array, chex.Array, chex.Array]:
+    if num_vehicles not in [2, 3, 4, 5]:
+        raise ValueError("num_vehicles must be 2, 3, 4 or 5 for num_customers=50")
+    map_max = 20
+    max_capacity = 300
+    max_start_window = 40.0
+    early_coef_rand = (0.0, 0.2)
+    late_coef_rand = (0.0, 1.0)
+    if num_vehicles == 2:
+        customer_demand_max = 10
+    elif num_vehicles == 3:
+        customer_demand_max = 15
+    elif num_vehicles == 4:
+        customer_demand_max = 20
+    else:
+        customer_demand_max = 25
+    return (
+        map_max,
+        max_capacity,
+        max_start_window,
+        early_coef_rand,
+        late_coef_rand,
+        customer_demand_max,
+    )
+
+
+def get_150_customer_init_settings(
+    num_vehicles: int,
+) -> Tuple[chex.Array, chex.Array, chex.Array, chex.Array, chex.Array, chex.Array]:
+    if not num_vehicles == 5:
+        raise ValueError("num_vehicles must be 5 for num_customers=50")
+    map_max = 20
+    max_capacity = 180
+    max_start_window = 60.0
+    early_coef_rand = (0.1, 0.1)
+    late_coef_rand = (0.5, 0.5)
+    customer_demand_max = 10
+
+    return (
+        map_max,
+        max_capacity,
+        max_start_window,
+        early_coef_rand,
+        late_coef_rand,
+        customer_demand_max,
+    )
+
+
+def get_init_settings(
```

Comment:
This function seems rather unflexible. I think the `num_customers` argument should be passed to the generator. In the current state, a user cannot user any number that is outside of the list below.

Reply:
Could we maybe discuss this over a call? I am not sure if I understand what you mean?

Reply:
I updated this as requested :+1: